core/data-api/src/main/java/com/dev/innerview/core/data_api/InnerViewRepository.kt

```diff
@@ -0,0 +1,12 @@
+package com.dev.innerview.core.data_api
+
+import com.dev.innerview.core.model.InnerView
+import com.dev.innerview.core.model.InnerViewType
+import kotlinx.coroutines.flow.Flow
+
+interface InnerViewRepository {
+
+    fun getInnerViews(): Flow<List<InnerView>>
+
+    suspend fun addInnerView(title: String, type: InnerViewType)
```

Comment:
🚫 **[ktlint]** <standard:function-signature> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should start on a newline
<!-- __reviewdog__:ChBhODYzOTc0MmMzNDYyYzRlEgZrdGxpbnQ= -->